### DISCUSSION TITLE
net8-windows aka Modernized UWP

### DISCUSSION THREAD

**Discussion by devsko** [CommunityToolkit/Windows#530] [Q&A] [answered]
Do you plan to publish an updated toolkit version for the net8 UWP platform? As far as i can see there is already a [branch](https://github.com/CommunityToolkit/Windows/tree/uwp-net8-windows) that seem to depend on an open WinUI2 [PR](https://github.com/microsoft/microsoft-ui-xaml/pull/9976).
I would *love* to update some UWP stuff!

**Comment by Arlodotexe** [comment, marked as answer]
Yes, we've been preparing to support modern UWP in the Windows Community Toolkit. We're hoping to release this alongside 8.2 once we get support from our dependencies!
